common/lib/xmodule/xmodule/js/src/sequence/display.js

```diff
@@ -448,6 +451,23 @@
             this.el.find('.nav-item.active .bookmark-icon-sr').text('');
         };
 
+        /**
+         * Sends the vertical+block id to Google Analytics,
+         * in order to track by unit item.
+         */
+        Sequence.prototype.sendPageViewToGA = function(position) {
+            var element = this.link_for(position);
+            var usageKey = element[0].attributes['data-id'].value;
```

Comment:
Could you ensure that the element length  is longer than 0 and that the value returned is valid ?

Reply:
See 415 line. @andrey-canon